prow/cmd/build/controller_test.go

```diff
@@ -72,6 +79,18 @@ func (r *fakeReconciler) getProwJob(name string) (*prowjobv1.ProwJob, error) {
 	return &pj, nil
 }
 
+func (r *fakeReconciler) terminateDupProwJobs(ctx string, namespace string) error {
+	return nil
+}
+
+func (r *fakeReconciler) getProwJobs() ([]prowjobv1.ProwJob, error) {
+	jobs := []prowjobv1.ProwJob{}
+	for _, j := range r.jobs {
```

Comment:
```golang
var jobs []prowjobv1.ProwJob
copy(r.jobs, jobs)
return jobs
```

Reply:
The copy cannot be used directly here because r.jobs is a map not a slice.